templates/cwr-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=CWR Server
+After=network.target
+After=systemd-user-sessions.service
+
+[Service]
+User=ubuntu
+Type=simple
+Restart=always
+WorkingDirectory={{charm_dir}}
+Environment=PYTHONPATH=${PYTHONPATH}:./lib
+Environment=FLASK_APP=./lib/CIGWServer.py
```

Comment:
I had to add these two lines as well:

Environment=LC_ALL=C.UTF-8
Environment=LANG=C.UTF-8